DemoApp/DemoApp/DemoApp/DTOs/TeacherDtoWithoutId.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace DemoApp.DTOs
+{
+    public class TeacherDtoWithoutId
+    {
+        public int YearsOfExperience { get; set; }
```

Comment:
Why only YearsOfExperience?

Reply:
added a new dto which include person details when a student is requested